packages/jest-snapshot/src/__tests__/snapshot.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
```

Comment:
oh, btw. these need the @oncall notation that other tests have as well :D Sorry we don't have a lint rule for this.